react/week-1/learning-objectives.md

```diff
@@ -18,14 +18,9 @@
 ### Passing props
 
 - Pass values to components as React props
-  - Receive props in a component (both function and class components)
+  - Receive props in a function component
   - Pass functions as props and call them in child components
 
 ### Deployment
 
 - Build and deploy a React application (to Netlify/Now/Heroku/etc)
```

Comment:
Is the plan to do the bare minimum to use React (eg https://kentcdodds.com/blog/super-simple-start-to-react), or perhaps something like create-react-app?